### DISCUSSION TITLE
Monster corpses keep moving around after death in multiplayer

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15458] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/user-attachments/assets/3fe88dfb-712d-48ca-b508-700da88cd68b

### Reproduction steps

.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
the issue seems has potentially been fixed in latest unstable

**Reply by Regalis11** [reply]
Yeah, this should be addressed in the unstable version, and the fix will be included in the next update. Sorry for the inconvenience!
